src/contracts/UserSetting/ValueDefinitionGroupInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\User\UserSetting;
+
+/**
+ * Interface for displaying User Preferences in the Admin UI.
+ *
+ * User Preferences are not displayed by default unless
+ * ValueDefinitionInterface implementation is provided.
+ */
+interface ValueDefinitionGroupInterface
+{
+    /**
+     * Returns name of a User Preference Group displayed in UI.
```

Comment:
Nitpick: Maybe we should stick to `User Setting` name to avoid mixing `setting` and `preference`?